utils/setup_build_env.sh

```diff
@@ -14,3 +14,10 @@ python3 -m pip install --requirement requirements.txt
 
 # Install required R package
 Rscript -e "install.packages('survey')"
+
+export $(cat .env | sed 's/#.*//g' | xargs)
```

Comment:
just noting in case you like it: I stumbled upon and used [this approach](https://github.com/NYCPlanning/aue/blob/master/.devcontainer/devcontainer.json#L31-L33) to using a local env file in a dev container